freeze_dependencies.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+def main():
+    os.system("pip freeze > frozen_dependencies.txt")
```

Comment:
Just out of curiosity, why define a separate .py file for this instead of directly defining the `entry` of https://github.com/catalystneuro/datta-lab-to-nwb/pull/58/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R17 to be the `pip freeze` command?

Reply:
It's my first time messing around with pre-commit, so maybe there is an easier way to accomplish this.  But I just tried replacing `entry: python -m freeze_dependencies` with `entry: pip freeze > frozen_dependencies.txt`, and it didn't work (pre-commit said the hook passed but it didn't actually write the file).